Add documentation page and banner to new users (#113)
sbv-world-health-org-metrics/sbv-world-health-org-metrics: who-metrics-ui/docs/definitions.md

## Welcome to the Open-Source Metrics Documentation

#### How to Read World Health Organization Open-Source Dashboard

The Dashboard is currently a snapshot of data from the **World Organization Health** Repositories. All the metrics are based on the _Last Updated Date_ reflected on the top of the dashboard.

### Metrics and Definitions

`Collaborators:` Collaborator is an individual that can read a write to the current repository. Metric calculates the current number of Collaborators.

`Watchers:` The number of users who are tracking a particular repository, receiving notifications regarding its activity.

`Open Issues:` Issues are tickets that represent bugs or new features that other people collaborate. Open issues are total of Open issues.

`Open PR's:` A pull request is a request to make a change to the master code of the project. Open PR's are the total of pull requests with status Open.

`Merge PR's`: A total of Pull Requests that are completed into the code based of the project.

> `Total Fork:` Fork is copy of the current repository into other repositories. Total Forks is how many times the repository has been copied.

Reviewer: ```suggestion
`Total Fork:` A fork is a copy of the current repository belonging to another user. Total Forks is how many times the repository has been copied.
```
